vwr/driver.py

```diff
@@ -66,10 +84,15 @@ def turn_off(self):
 
     def get(self):
         """Gets the setpoint and internal temperature."""
-        return {'setpoint': self.get_setpoint(),
-                'actual': self.get_internal_temperature(),
-                'pump': self.get_pump_speed(),
-                'on': self.get_operating_status()}
+        response = {'setpoint': self.get_setpoint(),
+                    'actual': self.get_internal_temperature(),
+                    'pump': self.get_pump_speed(),
+                    'on': self.get_operating_status()}
```

Comment:
You could just put `'connected': True` in the `response` variable. It only gets returned if connected anyway.